dev-lang/rust/rust-1.41.1.ebuild

```diff
@@ -272,14 +272,18 @@ src_configure() {
 }
 
 src_compile() {
+	IFS=$'\n'
```

Comment:
I would suggest to move call to `env` to subshell, and change `IFS` only in this subshell.
`$(cat "${S}"/config.env)` can be simplified to `$(<"${S}"/config.env)`.
```
src_compile() {
	(
		IFS=$'\n'
		env $(<"${S}"/config.env) ...
	)
```